src/components/Product.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Recent kun je zelfs deze import zin weglaten. Dat is nu standaard.
Laten staan is ook prima.